packages/main/src/themes/CheckBox.css

```diff
@@ -230,7 +230,7 @@
 	font-size: inherit;
 }
 
-.ui5-checkbox-root .ui5-checkbox-label {
+:host([wrapping-type="None"]) .ui5-checkbox-root .ui5-checkbox-label {
```

Comment:
Hi @yanaminkova,

Upon review, I noticed that the space between the checkbox and its surrounding text has disappeared. Is this the desired look? It appears strange, especially on items with `wrapping-type="Normal"`. Additionally, I observed that the cursor changes upon hovering over the text.

<img width="2560" alt="Screenshot 2024-06-12 at 17 23 16" src="https://github.com/SAP/ui5-webcomponents/assets/68374332/e7b141bd-3afb-4fea-9465-f47f9bfdbc7b">